greenplumpython/experimental/embedding.py

```diff
@@ -0,0 +1,174 @@
+from collections import abc
+from typing import Any
+from uuid import uuid4
+
+import greenplumpython as gp
+
+_vector_type = gp.type_("vector", modifier=384)
+
+
+@gp.create_function
+def _generate_embedding(content: str, model_name: str) -> _vector_type:
+    import sys
+
+    SD = globals().get("SD", sys.modules["plpy"]._SD)
```

Comment:
`sys.modules["plpy"]._SD` this could occurs error because of dill

`ExternalRoutineException: AttributeError: module 'plpy' has no attribute '_SD'  (seg1 10.128.0.23:8001 pid=3323603)`